ophyd/v2/_channelsim.py

```diff
@@ -23,9 +24,14 @@ def make_sim_descriptor(source: str, value) -> Descriptor:
         dtype = primitive_dtypes[type(value)]
         shape = []
     except KeyError:
-        assert isinstance(value, Sequence), f"Can't get dtype for {type(value)}"
-        dtype = "array"
-        shape = [len(value)]
+        if isinstance(value, Sequence):
+            dtype = "array"
+            shape = [len(value)]
+        elif issubclass(type(value), Enum):
+            dtype = "enum"
+            shape = []
```

Comment:
@danielballan @tacaswell event model currently has no concept of an Enum. Should we represent as a string, or extend event model? I don't know if downstream should care if it is an Enum as event docs are read-only, so it will never have to set it. The `get_value` and `set` methods in ophyd support Enum types

Reply:
Decided on `dtype=string` but add `choices=[""one","two"]` into descriptor